[FIX] mail: field labels of mail.link.preview model

niyasraphy opened the pull request:
**Description of the issue/feature this PR addresses:**

Currently no form view is defined for the model **mail.link.preview and mail.message.reaction** and field labels where in small letters. So the UI is not looking good as odoo renders dynamic form view.

**Current behavior before PR:**
Missing form view for multiple models. And for the mail.message.reaction model, all the fields are readonly fields, so there is no point of showing the create and edition buttons.

![Screenshot from 2022-10-21 11-30-23](https://user-images.githubusercontent.com/27989791/197152224-eb8a1375-b02b-4206-8dd0-51339cee70fd.png)

**Desired behavior after PR is merged:**
Clean form view

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/103732).

niyasraphy commented:
@IT-Ideas this is a similar topic that we discussed in another thread.
What about this form views, that opens from menus ?
Looking for your thoughts and suggestions

IT-Ideas commented:
> @IT-Ideas this is a similar topic that we discussed in another thread.
> What about this form views, that opens from menus ?
> Looking for your thoughts and suggestions

Hey @niyasraphy this is not my scope. I'll leave it to the dedicated team (see reviewers).

seb-odoo commented:
Hello, runbot is not happy with adding these views in stable, but it should be fine in master.

It's ok to keep this PR to add the capital letters to words, but .pot file should be updated as well.

niyasraphy commented:
@seb-odoo thanks for the feedback and will update the pr as per the given suggestion

niyasraphy commented:
@seb-odoo as per the suggestion, updated the PR with po files

seb-odoo commented on addons/mail/models/mail_link_preview.py:
```diff
@@ -15,13 +15,13 @@ class LinkPreview(models.Model):
+    og_title = fields.Char('Title')
+    og_image = fields.Char('Image')
+    og_description = fields.Text('Description')
+    og_mimetype = fields.Char('Mime Type')
+    image_mimetype = fields.Char('Image Mime Type')
```
I would suggest using `MIME type` for these.

niyasraphy replied:
@seb-odoo updated

seb-odoo replied:
it's not updated according to suggestion

niyasraphy replied:
@seb-odoo thanks again, updated the fields label as per suggestion

seb-odoo commented on addons/mail/models/mail_link_preview.py:
```diff
@@ -15,13 +15,13 @@ class LinkPreview(models.Model):
-    og_description = fields.Text('description')
-    og_mimetype = fields.Char('mimetype')
-    image_mimetype = fields.Char('image_mimetype')
+    source_url = fields.Char('Url', required=True)
```
I would suggest `URL` for this one.

niyasraphy replied:
updated

seb-odoo commented on addons/mail/i18n/mail.pot:
```diff
@@ -4,10 +4,10 @@
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-10 10:22+0000\n"
-"PO-Revision-Date: 2022-10-10 10:22+0000\n"
+"POT-Creation-Date: 2022-10-26 13:41+0000\n"
+"PO-Revision-Date: 2022-10-26 13:41+0000\n"
```
We usually don't update these meta data in the diff as they are not really useful and when updated they are annoying to manage when there are conflicts between PRs.

seb-odoo replied:
This wasn't updated either.

seb-odoo reviewed (commented):
Note: commit title could be a bit more specific, at least specifying which model is concerned

niyasraphy commented:
@seb-odoo updated pr title/commit message and  suggested changes

seb-odoo commented:
I'm not seeing all the changes in the new diff, only some of them?

niyasraphy commented:
@seb-odoo i have removed view related changes from this PR and created separate PR for the master.
and in this pr, updated the field labels, as you suggested(URL and Mime type) along with po files,
did i miss anything, let me know.
Thanks

seb-odoo reviewed (approved):
Thank you for the update

@robodoo r+
